tests/integ/modin/series/test_sample.py

```diff
@@ -24,15 +24,16 @@ def ignore_index(request):
 def test_series_sample_n(n, ignore_index):
     s = pd.Series(range(100, 110)).sample(n=n, ignore_index=ignore_index)
     assert len(s) == (n if n is not None else 1)
-    assert_index_equal(s.index, s.index)
+    assert_index_equal(s.index.to_pandas(), s.index.to_pandas())
 
 
 @pytest.mark.parametrize("frac", [None, 0, 0.1, 0.5, 0.8, 1])
-@sql_count_checker(query_count=4)
+@sql_count_checker(query_count=3)
 def test_series_sample_frac(frac, ignore_index):
     s = pd.Series(range(100, 110)).sample(frac=frac, ignore_index=ignore_index)
-    assert s.index.is_unique
-    assert_index_equal(s.index, s.index)
+    index = s.index
+    assert index.is_unique
+    assert_index_equal(index.to_pandas(), s.index.to_pandas())
```

Comment:
I believe you don't need to change these lines - it should still work as expected regardless of whether it's native pandas Index or lazy Index.

Reply:
`assert_index_equal` is a native pandas method, so this test fails without the conversion to native pandas

Reply:
can you add a todo for create our own `assert_index_equal`?

Reply:
created this function and replaced it in our tests